### DISCUSSION TITLE
Spawn entity for player(s)

### DISCUSSION THREAD

**Discussion by 2002Spiele** [ShaneBeee/SkBee#390] [Ideas]
Good day, Shane
I'd love if there would be an effect in SkBee which spawns an entity that's only visible to the specified players, which might also try to keep it invisible to other players. (entities aren't shown for players anymore after being far enough, but would load them again after going back)
My idea for the syntax would be `spawn [%number% of] %entitytypes% [%directions% %locations%] [with [nbt] %nbtcompound%] for %players%`.
I get really annoyed having to install another skript addon for packets just to make an entity invisible to all players except one, and other people surely do too.
Vanilla Skript also has such thing for blocks (`make %players% see %blocks% as %itemtype/blockdata%`) and it'd be nice to add the same thing for entities.
It would also be nice if you add an effect which does the same, but it does it to already existing entities.
Thank you for reading and I hope you consider adding this effect. :+1:

**Comment by ShaneBeee** [comment]
for suggestions, please plop them in the issue tracker so I can keep track of them.

**Comment by 2002Spiele** [comment]
oh alright
